vdev/src/testing/runner.rs

```diff
@@ -129,6 +129,32 @@ pub trait ContainerTestRunner: TestRunner {
         Ok(RunnerState::Missing)
     }
 
+    fn labels(&self) -> Result<BTreeMap<String, String>> {
```

Comment:
Haven't tested this yet fully, hence this is marked as a draft. However I am curious if the general step of reading from `command.check_output().lines()` is what we expect.

Would also like suggestions on what the return value should be